src/opnsense/mvc/app/controllers/OPNsense/Unbound/forms/advanced.xml

```diff
@@ -105,6 +105,18 @@
             slow queries or high query rates.
         </help>
     </field>
+    <field>
+        <id>unbound.advanced.discardtimeout</id>
+        <label>Discard Timeout</label>
+        <type>text</type>
+        <help>
+	    The wait time in msec where recursion requests are dropped. This is to stop a large number of replies
+            from accumulating.
+            If 'Serve Expired Responses' is enabled it should be set greater than 'Client Expired Response Timeout'.
+            Otherwise, a value of 1900 msec is suggested.
+            The value 0 disables it. Default 1900 msec.
```

Comment:
"0" could be problematic which may be indistinguishable from "" (the empty default), but I leave that for @swhite2 to check

Reply:
I know serve-expired-ttl (which we use) has a similar use of 0 as per the [unbound doc](https://nlnetlabs.nl/documentation/unbound/unbound.conf/)

I just checked - and if I leave the value blank in the opnsense UI, no value is written to the configuration for discard-timeout. Once I set a value, it is written. Finally, if I specifically put 0 there, that does get written.

So I think in all cases it matches the unbound docs ?